spec.html

```diff
@@ -15463,14 +15557,18 @@ <h2>Syntax</h2>
 
         ObjectBindingPattern[Yield, Await] :
           `{` `}`
+          `{` BindingRestProperty[?Yield, ?Await]? `}`
```

Comment:
These two lines are ambiguous; it would be fine to either remove the ? Or delete the first of the two lines.